### DISCUSSION TITLE
Allow exluding paths in the esbuild plugin

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [vanilla-extract-css/vanilla-extract#212] [Ideas]
I have a `stories` folder in my `src` for storybook. I'd like the esbuild plugin to allow me to exclude that directory. It might read what folder using the esbuild `external` config option.

**Comment by mattcompiles** [comment]
What do you mean by `exclude`? esbuild is just walking your imports from your entrypoints and if it sees a `.css.ts` file, processes it through vanilla-extract. Could you clarify your use-case more?

**Reply by alvarlagerlof** [reply]
The issue seems to have gone away by itself.
